hbase-common/src/main/resources/hbase-default.xml

```diff
@@ -2062,4 +2062,9 @@ possible configurations would overwhelm and obscure the important.
       hitting the namenode, if the DFSInputStream's block list is incomplete.
     </description>
   </property>
+  <property>
+    <name>hbase.fs.reader.warn.time</name>
```

Comment:
Better add a '.ms' suffix so users know it is in milliseconds, without reading the description.

Reply:
fine, added it